libr/bin/format/pe/pe.c

```diff
@@ -778,6 +801,39 @@ static int bin_pe_init_hdr(RBinPEObj* pe) {
 		pe_printf ("Warning: read (nt header)\n");
 		return false;
 	}
+	
+	// Enhanced PE signature validation
```

Comment:
there's a new check inside this function that makes the tests fail. try using #if 0/#endif to comment them until the tests pass. you can run few tests with `r2r test/db/path/to/test`